app/code/Magento/Cron/Console/Command/CronInstallCommand.php

```diff
@@ -76,4 +87,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return Cli::RETURN_SUCCESS;
     }
+
+    /**
+     * Returns an array of crons with "default" cron group
+     *
+     * @param array $crons
+     * @return array
+     */
+    private function extractDefaultCrons(array $crons = []): array
+    {
+        $defaultCrons = [];
+
+        if (count($crons)) {
```

Comment:
This `count` check is not necessary. We can be sure the variable is an array as its enforced by the typehint in the argument. And if the array is empty, the foreach will not do anything. So you can drop the `count` check here.

Reply:
**done**

Thanks